### DISCUSSION TITLE
Create tickets from admin portal for manual orders

### DISCUSSION THREAD

**Discussion by rickcj** [pretix/pretix#4041] [Ideas]
We came from TicketSource to Pretix (mainly due to the relatively high fee's for our low value (£2-£6) tickets).

We were able to process ticket orders from the admin, which means you can use manual payment methods, but disable those payment methods from the customer side.

Would be useful to add the ability to create tickets purchased in person through the admin, instead of as a regular customer and allow it to be marked as paid during checkout instead of having to go into admin to manually update the order as paid.

**Comment by pc-coholic** [comment]
> Would be useful to add the ability to create tickets purchased in person through the admin, instead of as a regular customer and allow it to be marked as paid during checkout instead of having to go into admin to manually update the order as paid.

This is an often requested feature that might make its way into pretix at some point in the future - but no promises yet :-)

As of right now, the way to go would be to
a) use pretixPOS: preferred - and even mandatory in a lot of jurisdictions due to fiscalisation laws
b) use the pretix-resellers module
